src/verification/utils/definitions/disableNl.gobra

```diff
@@ -0,0 +1,137 @@
+// Copyright 2022 ETH Zurich
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +gobra
+
+package definitions
+
+// lemmas for working with disableNL
+
+ghost
+requires a == b
+ensures a * c == b * c
+ensures c * a == c * b
+decreases
+func LemmaMulReplace(a, b, c int) { }
+
+ghost
+requires a == b
+requires c == d * a
+ensures  c == d * b
+decreases
+func LemmaMulSubst1(a, b, c, d int) { }
```

Comment:
Is there a `LemmaMulSubst2`? Otherwise, let's drop the number. Please double check this everywhere else